spacialgaze-plugins/SG.js

```diff
@@ -116,6 +116,22 @@ SG.reloadCSS = function () {
 	http.get(options);
 };
 
+//Daily Rewards System for SpacialGaze by Lord Haji
+SG.giveDailyReward = function (userid, user) {
+	if (!user || !userid) return false;
+	userid = toId(userid);
+	if (!user.autoconfirmed) return false;
```

Comment:
We might want to include non-autoconfirmed users in this, but thats a policy decision too so don't change anything yet.